<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-27T10:03:53 IST" name="Suite" finished-at="2025-06-27T10:04:22 IST" duration-ms="29173">
    <groups>
    </groups>
    <test started-at="2025-06-27T10:03:53 IST" name="chromeTestLogin" finished-at="2025-06-27T10:04:22 IST" duration-ms="29173">
      <class name="TestCases.LoginTest">
        <test-method is-config="true" signature="extentReportMethod()[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:03:53 IST" name="extentReportMethod" finished-at="2025-06-27T10:03:53 IST" duration-ms="754" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentReportMethod -->
        <test-method is-config="true" signature="initialization(java.lang.String,java.lang.reflect.Method)[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:03:54 IST" name="initialization" finished-at="2025-06-27T10:04:07 IST" duration-ms="13103" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestCases.LoginTest.loginTestCase()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method retried="true" signature="loginTestCase()[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:07 IST" name="loginTestCase" finished-at="2025-06-27T10:04:07 IST" duration-ms="13" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "utills.suiteListener.extent" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "utills.suiteListener.extent" is null
at utills.suiteListener.onTestStart(suiteListener.java:57)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:103)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:569)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestCase -->
        <test-method is-config="true" signature="onTestFailure(org.testng.ITestResult)[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:07 IST" name="onTestFailure" finished-at="2025-06-27T10:04:07 IST" duration-ms="437" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTestCase status=SKIP method=LoginTest.loginTestCase()[pri:0, instance:TestCases.LoginTest@70e9c95d] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onTestFailure -->
        <test-method is-config="true" signature="initialization(java.lang.String,java.lang.reflect.Method)[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:07 IST" name="initialization" finished-at="2025-06-27T10:04:20 IST" duration-ms="13334" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void TestCases.LoginTest.loginTestCase()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method signature="loginTestCase()[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:20 IST" name="loginTestCase" finished-at="2025-06-27T10:04:20 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "utills.suiteListener.extent" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "utills.suiteListener.extent" is null
at utills.suiteListener.onTestStart(suiteListener.java:57)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:103)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:569)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestCase -->
        <test-method is-config="true" signature="onTestFailure(org.testng.ITestResult)[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:21 IST" name="onTestFailure" finished-at="2025-06-27T10:04:21 IST" duration-ms="318" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTestCase status=FAILURE method=LoginTest.loginTestCase()[pri:0, instance:TestCases.LoginTest@70e9c95d] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onTestFailure -->
        <test-method is-config="true" signature="onFinish(org.testng.ITestContext)[pri:0, instance:TestCases.LoginTest@70e9c95d]" started-at="2025-06-27T10:04:21 IST" name="onFinish" finished-at="2025-06-27T10:04:22 IST" duration-ms="777" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@39f5b723]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onFinish -->
      </class> <!-- TestCases.LoginTest -->
    </test> <!-- chromeTestLogin -->
  </suite> <!-- Suite -->
</testng-results>
